scripts/ci/operator.sh

```diff
@@ -0,0 +1,184 @@
+#!/bin/bash
+# shellcheck disable=SC2119
+
+##### subscription
+CATALOG_SOURCE="${CATALOG_SOURCE:-redhat-operators}"
+SUBSCRIPTION_NAMES=("jaeger-product" "kiali-ossm" "servicemeshoperator")
+
+createSubscription() {
+  local name=$1
+
+  echo "Create Subscription resource for $name"
+  echo
+  kubectl apply -f - <<EOF
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: "${name}"
+  namespace: openshift-operators
+spec:
+  channel: stable
```

Comment:
What happens when someone submits a PR containing a test that only passes against a pre-release version of the operator. How will we make this script run against this version of the operator?

Reply:
Nevermind. I talked to Filip about this and it's fine as is.